web/app/scripts/views/dashboard/dashboard-partial.html

```diff
@@ -39,6 +39,16 @@
     </div>
 
     <div ng-if="!ctl.showBlackSpots" class="blackspots-disabled">
+        <div class="block block-2x1 quick-map">
+            <div class="block-inner">
+                <div class="map" leaflet-map recent-events zoom-to-boundary></div>
+                <h3><span>{{ ctl.recordType.plural_label }}: {{ 'DASHBOARD.RECENT_EVENTS_PERIOD' | translate }}</span>
+                    <a ui-sref="map" class="small pull-right">
+                        {{ 'DASHBOARD.VIEW_ALL' | translate }} {{ ctl.recordType.plural_label | lowercase }} »
```

Comment:
I don't think so. It's been like this since the begininning and we haven't seen any problems with browsers we've tested with. Though I wouldn't doubt if an old browser might have an issue with it.